code/modules/transport/pods/warp_travel.dm

```diff
@@ -144,4 +173,179 @@ var/global/list/warp_beacons = list() //wow you should've made one for warp beac
 		playsound(src.loc, "warp", 50, 1, 0.2, 1.2)
 		do_teleport(M, src.target, 1) ///You will appear adjacent to the beacon
 
+/obj/warp_beacon/proc/startpack()
+	src.packable = 0
+	src.icon_state = "beaconpack"
+	SPAWN_DBG(14) //wait until packing is complete
+		var/obj/beacon_deployer/packitup = new /obj/beacon_deployer(src.loc)
+		playsound(get_turf(src), "sound/machines/heater_off.ogg", 20, 1)
+		if(src.beaconid)
+			packitup.beaconid = src.beaconid
+			packitup.name = "warp buoy unit [beaconid]"
+		SPAWN_DBG(0)
```

Comment:
I doubt this spawn is needed...

Reply:
Ingame test confirmed this. Removed it as well as its counterpart in the deployment proc.